server/middleware/token.js

```diff
@@ -96,7 +96,7 @@ function token(options) {
     var registry = app.registry;
     if (!TokenModel) {
       if (registry === loopback.registry) {
-        TokenModel = options.model || loopback.AccessToken;
+        TokenModel = app.models[options.model] || loopback.AccessToken;
```

Comment:
yes, i thought that too
**simplifying** was my next move: 

i think we can even simplify to just this:
``` 
TokenModel = registry.getModel(options.model || 'AccessToken'); 
```
as it does the exact same thing, thoughts?